### PR TITLE
create contact type & contact models

### PR DESCRIPTION
**Pull request by gluuck**
created two models contact and contact type

### CODE REVIEW

File: app/models/contact.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Contact < ApplicationRecord
```
Comment (XaoGao) [inline app/models/contact.rb:1]: @gluuck call gem annotate

File: app/models/contact_type.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class ContactType < ApplicationRecord
```
Comment (XaoGao) [inline app/models/contact_type.rb:1]: @gluuck add validations for 'name' and 'icon'

File: spec/models/contact_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (XaoGao) [inline spec/models/contact_spec.rb:1]: @gluuck double quotes

**Review by XaoGao — COMMENTED** [review]
@gluuck pls fix

**Comment by XaoGao** [comment]
@gluuck before you have to write me in issue: "i wanna take this issue" and wait my answer, pls read README file

**Comment by XaoGao** [comment]
@gluuck не прошли тесты и rubocop

**Comment by gluuck** [comment]
@XaoGao опечатка была

**Pull request merged**